File: 005_For_Loop_in_Advance/11_prime_no.cpp
// check a number is Prime Number  or not  

#include<iostream>
using namespace std;

int main()
{
    int n,i;
    cout<<"Enter number:";
    cin>>n;

    if(n<2)
    {
        cout<<"Not a Prime number.";
        return 0;
    }

    else
    {
        for(i=2; i<n; i++)
        {
            if(n%i==0)
            {
                cout<<"Not a Prime number.";
                return 0;
            }
        }
    }
    cout<<"Prime number.";
}

File: 5 For Loop in Advance/11_prime_no.py
num = int(input("Enter a number: "))

if num < 2:
    print(f"Not a prime numbers: {num}")
else:
    for i in range(2, num):
        if num%i==0:
            print(f"{num} is not a prime number")
            break
    else:      # "for-else" special Python feature
        print(f"{num} is a prime number")
    